google/resource-snippets/cloudkms-v1/alternatives/tf/cloudkms.tf

```diff
@@ -0,0 +1,45 @@
+variable "project_id" {}
+variable "deployment" {}
+
+provider "google" {
+  project = var.project_id
+  region  = "us-central1"
+  zone    = "us-central1-c"
+}
+
+locals{
+    user = "user:zyanshu@google.com"
+}
```

Comment:
This should be a user-configurable like in [deploymentmanager-samples/kms.yaml](https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/google/resource-snippets/cloudkms-v1/kms.yaml#L25), so can we use a variable instead of a local?